contracts/VotingMachines/GenesisProtocol.sol

```diff
@@ -80,7 +80,7 @@ contract GenesisProtocol is IntVoteInterface,UniversalScheme {
         mapping(address=>Staker) stakers;
     }
 
-    event ExecuteProposal(bytes32 indexed _proposalId,
+    event GPExecuteProposal(bytes32 indexed _proposalId,
```

Comment:
the GPExecuteProposal should include only the extended params which are not included on ExecuteProposal common interface (with the proposalId as Index).

Reply:
Done

Reply:
Does the index  _avatar is really needed here ?
One can fetch the _proposalId via the _avatar from ExecuteProposal .

Reply:
> Does the index _avatar is really needed here?

I left it in to maintain compatibility with all of the other GP events, as well as all of the events in AbsoluteVote and IntVoteInterrface and almost every other scheme event in Arc.

The only inconsistencies are a handful of events in VestingScheme (five) and VoteInOrganizationScheme (one).  I think we should seriously consider adding indexed _avatar to those as well.

This kind of API consistency is very important in supporting consumers of any library or framework, including this one.

Reply:
Removed